subjects/mobile-dev/map-markers/README.md

```diff
@@ -1,38 +1,67 @@
 ## map-markers
 
-### Introduction
+In this project, your team will create an app that uses the `Google Maps API`. The app must allow users to save and display their favorite places on a map.
 
-Create an app where you can save the list of your favorite places and show them in the app.
+### Instructions
 
-In this raid, your team should implement an app which uses google maps.
+The app should have the following functionality:
 
-### Objective
+A `TabBar` with three screens:
 
-Implement following functionality:
+- A `Maps` screen like [google maps](https://codelabs.developers.google.com/codelabs/google-maps-in-flutter#0) where the user can see their favorite places marked on the map with an `info view`.
 
-- Tabbar app with 3 screens:
-  - Google maps screen [example](https://codelabs.developers.google.com/codelabs/google-maps-in-flutter#0)
-  - List of your favorite places.
-  - Info about your app. Names of the authors, their email adresses, and general description of what your application does.
+- A `favorite places` screen which will have a list of the user's favorite places.
```

Comment:
Minor formatting differences compared to the audit (`Favorite Places` in the audit)

Reply:
done